cop.h

```diff
@@ -558,6 +558,8 @@ string C<p>, creating the package if necessary.
 #define CopHINTHASH_get(c)	((COPHH*)((c)->cop_hints_hash))
 #define CopHINTHASH_set(c,h)	((c)->cop_hints_hash = (h))
 
+#define CopFEATURES_setfrom(c, o) ((c)->cop_features = (o)->cop_features)
```

Comment:
Tiniest of nitpics: isn't it traditional to use `dst` and `src` for these two arguments?

Reply:
This seems in keeping with the style of the file, and is harmless, even though generally id agree with you that it would be more comprehensible.

Reply:
Fixed.